### ISSUE TITLE
Doesn't work in mist

### ISSUE THREAD

**Issue by linagee** [web3-gov/meteor-dapp-boardroom#8]
Please make this work in mist. I load it up (using http://meteor-dapp-boardroom.meteor.com/) and it doesn't recognize any of my accounts even when I select them.

**Comment by SilentCicero** [comment]
Yep, the accounts system is internal. This product is ready for mist yet.
